Add release note for #860 (#937)
submariner-io/lighthouse: release-notes/20221102-no-records-dns.md

> Lighthouse now returns DNS error message in the response body when no matching records are found for the query to 
> "clusterset.local". This will prevent unnecessary retries.

Reviewer: ```suggestion
Service Discovery now returns a DNS error message in the response body when no matching records are found for the query to 
"clusterset.local". This prevents unnecessary retries.
```

<!-- markdownlint-disable MD041 -->

> Service Discovery now returns a DNS error message in the response body when no matching records are found for the query to 

Reviewer 2: There’s a trailing space at the end of this line:

```suggestion
Service Discovery now returns a DNS error message in the response body when no matching records are found for the query to
```
